components/menu/Menu.js

```diff
@@ -246,7 +246,7 @@ const factory = (MenuItem) => {
       }, this.props.className);
 
       return (
-        <div data-react-toolbox="menu" role="menu" className={className} style={this.getRootStyle()} onKeyDown={this.handleEscape} tabIndex={0}>
+        <div data-react-toolbox="menu" role="menu" className={className} style={this.getRootStyle()} onKeyDown={this.handleEscape} tabIndex={this.state.tabIndex}>
```

Comment:
instead of keeping tabIndex as state can you just check state.active and derive tabIndex based off of that here?